packages/atomic/src/components/atomic-refine-modal/atomic-refine-modal.tsx

```diff
@@ -220,14 +207,23 @@ export class AtomicRefineModal implements InitializableComponent {
     ];
   }
 
+  private renderBody() {
+    return (
+      <aside slot="body" class="flex flex-col w-full adjust-for-scroll-bar">
```

Comment:
I couldn't reproduce the case where `adjust-for-scroll-bar` is necessary. Any ideas?

Reply:
No clue!

Reply:
do you happen to know, @samisayegh ?